Package.swift

```diff
@@ -360,14 +385,16 @@ if ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] == nil {
         // used by 'swift-driver' and 'sourcekit-lsp'. Please coordinate
         // dependency version changes here with those projects.
         .package(url: "https://github.com/apple/swift-argument-parser.git", .upToNextMinor(from: "0.4.3")),
-        .package(url: "https://github.com/apple/swift-driver.git", .branch(relatedDependenciesBranch)),
+        .package(url: "https://github.com/apple/swift-collections.git", .branch("main")),
```

Comment:
this should be pinned to a stable version, I believe 0.0.3 is the one we want

Reply:
Done.

Also, I changed other dependencies' versions to use `Version`'s initialiser directly, instead of string literals. I think this reduces a tiny bit of dependency resolution work, but probably isn't necessary. I can change it back if string literals are preferred.